plutus-core/testlib/PlutusCore/Generators/QuickCheck/Utils.hs

```diff
@@ -62,7 +63,7 @@ assertNoCounterexamples bad = ceDoc (prettyPirReadable bad) False
 -- they relate to each other.
 -- | A type is a container for the purposes of shrinking if it has:
 class Container f where
-  data OneHoleContext f :: * -> *
+  data OneHoleContext f :: Kind.Type -> Kind.Type
```

Comment:
Must be qualified since `Type` is already in scope from some `Plutus` module

Reply:
We usually use the `GHC.` qualification instead of `Kind.`, but feel free to ignore this nitpicking.

Reply:
I'm happy to update it 😄